pyproject.toml

```
@@ -23,15 +23,14 @@ classifiers = [
   "Topic :: Scientific/Engineering",
 ]
 dependencies = [
-  "numpy",
-  "numpy <1.25.0.dev0; python_version < '3.9'",
-  "scipy",
-  "nibabel >=2.1",
-  "pandas >=0.23",
+  "numpy >=1.19",  # NEP29 + 1yr
+  "scipy >=1.5",   # Scientific Python SPEC 0 + 1yr
```

Alternative 2: We could require more recent scipy.

```suggestion
  "scipy >=1.6",   # Scientific Python SPEC 0 + 1yr
```